README.md

```diff
@@ -7,211 +7,336 @@
 [![Subreddit subscribers](https://img.shields.io/reddit/subreddit-subscribers/spiceai?style=social)](https://www.reddit.com/r/spiceai)
 [![Follow on Twitter](https://img.shields.io/twitter/follow/spiceaihq.svg?style=social&logo=twitter)](https://twitter.com/intent/follow?screen_name=spiceaihq)
 
-**Spice.ai** makes it easy for developers to build apps that learn and adapt by streamlining the use of machine learning (ML) in software. Combined with time-series data, developers can create applications that continuously improve using ML recommendations.
+## What is SpiceAI?
 
-Spice.ai takes a developer-first approach, and is focused on a fast, iterative, inner development loop, enabling developers to get started with ML in minutes instead of months.
-
----
+**Spice.ai** is an open source runtime environment that enables developers to leverage rapid access to time-series data. The Spice AI Runtime enables the best in class high performance queries for powering an array of data driven applications.
 
 ⚠️ **DEVELOPER PREVIEW ONLY** Spice.ai is under active **alpha** stage development and is not intended to be used in production until its **1.0-stable** release.
 
----
+## Quick Start
 
-📢 Read the Spice.ai announcement blog post at [blog.spiceai.org](https://blog.spiceai.org).
+SpiceAI uses Go and Rust. Be sure to have both of these installed before proceeding.
 
-📺 View a getting started walkthrough of Spice.ai in action [here](https://www.youtube.com/watch?v=DKBLjuAz_lI).
+**Install Rust:**
+https://www.rust-lang.org/tools/install
 
-💻 Learn about our [approach to building intelligent applications](https://blog.spiceai.org/posts/2021/11/05/making-apps-that-learn-and-adapt/) and not just "doing AI"
+**Install Go:**
+https://go.dev/doc/install
 
-## Features
+**Step 1.** Install the SpiceAI CLI:
 
-The Spice.ai runtime is written in Golang and Python and runs as a container or microservice. It's deployable to any public cloud, on-premises, and edge. It is configured with a simple manifest and accessed by HTTP APIs.
+```bash
+curl https://install.spiceai.org | /bin/bash
+```
 
-Spice.ai includes:
+**Step 2.** Choose a project name and initialize a new project with the `spice init` command:
 
-- A lightweight, portable ML runtime accessible by simple HTTP APIs, allowing developers to use their preferred languages and frameworks
-- A developer-friendly CLI
-- Simple, git-committable, configuration and code
+```bash
+spice init my_spiceai_project
+```
 
-### Community-Driven Data Components
+This creates a spicepod.yaml file in your directory.
 
-Spice.ai also includes a library of community-driven [data components](https://github.com/spiceai/data-components-contrib) for streaming and processing time series data, enabling developers to quickly and easily combine data with learning to create intelligent models.
+```yaml
+version: v1beta1
+kind: Spicepod
+name: my_spiceai_project
+```
```

Comment:
I don't think its necessary to show the yaml - this is valuable screen real estate (near the top of the README), let's not waste it.

Reply:
removed.